app/src/main/res/values/colors.xml

```diff
@@ -4,4 +4,11 @@
   <color name="colorPrimaryDark">#00574B</color>
   <color name="colorAccent">#D81B60</color>
   <color name="ic_launcher_background">#26A69A</color>
+
+  <!--  BASIC COLORS-->
```

Comment:
Nit on spacing: ``<!-- BASIC COLORS -->``

Ditto below.

Reply:
Done. added spaces at both ends of text